### PR TITLE
updating keyvault client id

### PR DESCRIPTION
**Pull request by trwalke**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+2.7.1
```
Comment (bgavrilMS) [inline changelog.txt]: It's possible that you are using the old "master". Please try to delete your master branch and fetch it from the remote again. Then reopen this PR.

Comment (trwalke) [reply changelog.txt]: @bgavrilMS  it was the PR itself. It defaults to master as the branch to merge into. I changed the target to archive/release2x and everything is good

Comment (bgavrilMS) [reply changelog.txt]: Ah ok, got it. So we want to patch MSAL 2.x with this fix?

Comment (bgavrilMS) [reply changelog.txt]: Oh sorry, I got confused, I see what you are doing. LGTM.

**Comment by bgavrilMS** [comment]
@trwalke  - can we get this in to stabilize releases 2x?

**Comment by trwalke** [comment]
> @trwalke - can we get this in to stabilize releases 2x?

@bgavrilMS Ill merge it in today, im still seeing errors on the selenium ui stuff so I need to update automation ids i think.

**Pull request merged**